src/async-components/views/dialogs/security/ExportE2eKeysDialog.tsx

```diff
@@ -16,45 +16,49 @@ limitations under the License.
 
 import FileSaver from 'file-saver';
 import React, { createRef } from 'react';
-import PropTypes from 'prop-types';
 import { _t } from '../../../../languageHandler';
 
 import { MatrixClient } from 'matrix-js-sdk/src/client';
 import * as MegolmExportEncryption from '../../../../utils/MegolmExportEncryption';
-import * as sdk from '../../../../index';
+import BaseDialog from "../../../../components/views/dialogs/BaseDialog";
 
-const PHASE_EDIT = 1;
-const PHASE_EXPORTING = 2;
-
-export default class ExportE2eKeysDialog extends React.Component {
-    static propTypes = {
-        matrixClient: PropTypes.instanceOf(MatrixClient).isRequired,
-        onFinished: PropTypes.func.isRequired,
-    };
+export enum ExportE2EKeysPhase {
+    Edit = 1,
+    Exporting = 2,
+}
```

Comment:
Any reason this is exported? the old phases weren't
Also given that they weren't exported we could go back to an automatic/incremented enum instead of manually specifying all values